build.ps1

```diff
@@ -226,7 +245,11 @@ function CreateZips([string] $runtimeSuffix) {
     Rename-Item "$privateSiteExtensionPath" "$siteExtensionPath\$extensionVersion"
     Copy-Item .\src\WebJobs.Script.WebHost\extension.xml "$siteExtensionPath"
     ZipContent $siteExtensionPath "$buildOutput\Functions.$extensionVersion$runtimeSuffix.zip"
+}
 
+function deleteDuplicateWorkers() {
+    Write-Host "Deleting workers directory: $privateSiteExtensionPath\32bit\workers" 
```

Comment:
This does not look right. Contents for build.ps1 in master branch: https://github.com/Azure/azure-functions-host/blob/master/build.ps1 are expected contents. 
For the last release, i made changes in the master branch so the commit i mentioned should in master already?

Reply:
This has been resolved with the updated build script in dev (which has been cherry-picked into this branch)